ref_builder/otu/modify.py

```diff
@@ -11,7 +13,41 @@
     Segment,
 )
 from ref_builder.repo import Repo
-from ref_builder.resources import RepoOTU
+from ref_builder.resources import RepoOTU, RepoSequence
+
+
+logger = get_logger("otu.modify")
+
+
+def exclude_accessions_from_otu(
+    repo: Repo,
+    otu: RepoOTU,
+    accessions: list[str],
+) -> None:
+    """Take a list of accessions and add them to an OTU's excluded accessions list."""
+    excluded_accessions = set()
+
+    # Using a set avoid duplicate accessions.
+    for accession in set(accessions):
+        excluded_accessions = repo.exclude_accession(otu.id, accession)
+
+    logger.debug(
+        f"Accessions currently excluded from fetches: {excluded_accessions}",
+        taxid=otu.taxid,
+        otu_id=str(otu.id),
+        name=otu.name,
+    )
```

Comment:
This could be much simplified without the logging, which seems superfluous to me. Probably, the function could be eliminated entirely.